### DISCUSSION TITLE
Search packages by target

### DISCUSSION THREAD

**Discussion by geekosaur** [flora-pm/flora-server#528] [Feature Requests]
It occurred to me, based on a discussion in #haskell:matrix.org, that it would be nice to be able to search for packages based on public targets such as executables or public sublibraries. As these are listed in the Cabal file, this should be relatively easy; but it'll probably require extra indexing beyond the package itself.

**Comment by tchoutri** [comment]
Closed by [FLORA-62](https://plane.sh/flora-pm/f5cfae69-05d7-46ff-94d5-c605ecbc2073?board=list&peekId=35f93f3f-f3d2-4a07-82c9-1f838da0a22c)
